src/domain/synthetics/trade/utils/decrease.ts

```diff
@@ -355,7 +355,7 @@ export function getDecreasePositionAmounts(p: {
 
     values.collateralDeltaUsd =
       // https://app.asana.com/0/1204313444805313/1207549197964321/f
-      leverageWithoutPnl !== undefined
+      leverageWithoutPnl !== undefined && leverageWithoutPnl !== 0n
```

Comment:
leverage should not equal zero though
maybe update the comment when/why this happens?